### ISSUE TITLE
Processing of protected land use types

### ISSUE THREAD

**Issue by jmemb** [JGCRI/demeter#38]
Dear Demeter Team:

When running Demeter-2.0 and reaching the step of reading the projected GCAM data from the attached file (adapted to match Demeter input format requirements):
[Reference.csv](https://github.com/JGCRI/demeter/files/14232020/Reference.csv)
I get the following warning:
WARNING:root:Land classes in allocation file but not in projected model data:  ['protectedgrassland' 'protectedshrubland' 'protectedunmanagedforest' 'protectedunmanagedpasture']

The file it is reading does include the protected categories for grassland, shrubland unmanaged forest and pasture. Any idea how can I make Demeter process these protected land use types?

Thank you in advance.

Best,
Jorge

**Comment by kanishkan91** [comment]
@jmemb Thanks for your question. This example of demeter aggregates Protected land types into the Unmanaged land. To break out the same, you have two options -

1. You can program demeter to downscale protected lands i.e. it will breakup the protected lands into unmanaged and protected. However, in this case, demeter may downscale it arbitrarily without respecting the actual spatial distribution of protected lands. This would only require changing some mapping files. 
2. The second option would be to actually introduce protected areas in the basemap. This would give demeter the information on where protected areas are located and it would downscale other land use and land cover accordingly. This may be the more appropriate option. 

I'm assuming you want to use demeter with a GCAM 6/7 run? If so we have a demeter workflow that downscales land while incorporating current protection constraints. I think it would be easier to explain on a call or over email. Kindly email kanishka.narayan@pnnl.gov. Happy to discuss further.

**Comment by kanishkan91** [comment]
A workflow for downscaling protected areas is available here- https://zenodo.org/records/10826464. 

@jmemb - Once you have this working on your machine, I will close this

**Comment by jmemb** [comment]
Yes, it is working now. Thanks a lot for your help, @kanishkan91!

**Comment by kanishkan91** [comment]
Closing this. Thanks!

**Comment by jmemb** [comment]
Hi, again!

I wanted to bring up two issues on the tabular units which which I have notice when using the workflow for downscaling protected areas:
1. Activating the option to save tabular spatial landcover as CSV does not seem to work no matter whether the output is set to be “fraction” or “sqkm”.
2. The description of the parameter to create a NetCDF file (save_netcdf_yr) seems to indicate that the output can only be produced for percentages by grid cell containing each land class, but outputs in square kilometers can also be produced.

Best,
Jorge

**Comment by kanishkan91** [comment]
@jmemb I have reopened the issue. I will fix this in the code

**Comment by kanishkan91** [comment]
Fixed
